test/e2e/connectivity_test.go

```diff
@@ -121,6 +125,114 @@ func (data *TestData) runPingMesh(t *testing.T, podInfos []PodInfo, ctrname stri
 	}
 }
 
+// runPodsOnWindows checks the Pods' connectivity after antrea-agent is restarted.
+// This case only run on Windows. The Pods in the test includes both host-network Pod and none host-network Pod,
+// because CNI on Windows is also responsible for the host-network Pod's networking as long as it is not using
+// host-process containers.
+func testWindowsPodConnectivityAfterAntreaRestart(t *testing.T, data *TestData) {
+	podInfos, cleanupFn := data.runPodsOnWindows(t)
+	defer cleanupFn()
+
+	data.runPingMesh(t, podInfos, toolboxContainerName, true)
+	nodeFlows := data.countOVSFlowsOnWindows(t, podInfos)
+	err := data.RestartAntreaAgentPods(defaultTimeout)
+	assert.NoError(t, err)
+
+	err = wait.PollUntilContextTimeout(context.Background(), 5*time.Second, 1*time.Minute, false, func(ctx context.Context) (done bool, err error) {
+		newNodeFlows := data.countOVSFlowsOnWindows(t, podInfos)
+		for node, flows := range nodeFlows {
+			newFlows, found := newNodeFlows[node]
+			if !found {
+				return false, nil
+			}
+			if len(newFlows) != len(flows) {
+				return false, nil
+			}
+		}
+		return true, nil
+	})
+	assert.NoErrorf(t, err, "The Openflow entries should be consistent after Antrea agent restarts")
+
+	data.runPingMesh(t, podInfos, toolboxContainerName, true)
+	time.Sleep(10 * time.Minute)
```

Comment:
I think you had this for debugging purposes, that's a very long sleep :)

Reply:
Yes, I will remove it after finish the debugging on the new e2e case.